src/calculateRentalCost.js

```diff
@@ -4,7 +4,24 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const MIDDLE_TERM = 3;
+
+  const MIDDLE_TERM_DISCOUNT = 20;
+  const LONG_TERM = 7;
+
+  const LONG_TERM_DISCOUNT = 50;
+  let totalAmount = days * PRICE_PER_DAY;
+
+  if (days >= LONG_TERM) {
+    totalAmount -= LONG_TERM_DISCOUNT;
+  }
+
+  if (days >= MIDDLE_TERM && days < LONG_TERM) {
+    totalAmount -= MIDDLE_TERM_DISCOUNT;
+  }
```

Comment:
Since the `if` block on lines 16-18 has a `return` statement, the code execution will exit if the condition is true, making the `else` unnecessary. You can remove the `else` and unconditionally check if `days` is greater than or equal to `MIDDLE_TERM` and less than `LONG_TERM`, as per the checklist item to prefer `if` with `return` over `if else` to simplify later conditions.